src/scenes/CreateEvent/store.js

```diff
@@ -325,6 +339,7 @@ export default class CreateEventStore {
   /**
    * Gets the average block time from the Insight API.
    */
+  @action
```

Comment:
averageBlockTime is not set to @observable

Reply:
Oh I think you are misunderstanding the `action` keyword:
https://mobx.js.org/refguide/action.html#when-to-use-actions

I mark action whenever the state changes.

Reply:
ah I thought you meant those to be observable, alright.